src/components/aspectRatio/index.stories.tsx

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import { Meta, StoryObj } from '@storybook/react';
 
-import Box from '../box';
+import { Box } from 'src';
 
-import AspectRatioComponent from './index';
+import { AspectRatio } from './index';
 
-const meta: Meta<typeof AspectRatioComponent> = {
-  title: 'Layout/Aspect ratio',
-  component: AspectRatioComponent,
-  args: {
-    children: <Box backgroundColor="red.500">I am in the box with a ratio</Box>,
-  },
-};
-export default meta;
+const meta: Meta<typeof AspectRatio> = {
+  title: 'Layout/Aspect Ratio',
+  component: AspectRatio,
 
-export const Overview: StoryObj<typeof AspectRatioComponent> = {
   args: {
     width: 300,
     ratio: 4 / 3,
+    children: 'box',
```

Comment:
Storybook complains if we pass a react component as an argument. This is because all arguments need to be serialisable to reconstruct the story state via query params. We can leverage `mapping` in the `argTypes` to circumvent that.

More on [dealing with complex values](https://storybook.js.org/docs/essentials/controls#dealing-with-complex-values)